instance/application.cfg.default

```diff
@@ -3,6 +3,8 @@ DEBUG = True
 SECRET_KEY = 'generate some random secret string'
 SQLALCHEMY_DATABASE_URI = 'postgresql://user:password@localhost/dbname'
 SERVER_NAME = '<the name and port number of the server (e.g.: 'myapp.dev:5000')>'
+LOCKOUT_PERIOD_MINUTES = 30
```

Comment:
You'll want to move these defaults to the [`BaseConfig` class in config.py](https://github.com/uwcirg/true_nth_usa_portal/blob/develop/portal/config/config.py#L42)

`application.cfg.default` (and `application.cfg`) ideally are for config values that are required (do not and should not have a default)

Reply:
I'll remember this next time.